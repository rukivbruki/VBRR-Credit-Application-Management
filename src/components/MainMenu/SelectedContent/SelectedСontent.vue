<template>
  <div style="display: flex; flex-wrap: wrap; flex-direction: row">
    <slot />
    <el-main>
      <div class="el-main-flexContainer">
        <Card
          v-for="(card, id) in getCardsToShow"
          :key="card.name"
          :name="getCardsToShow[id].name"
          :links="getCardsToShow[id].links"
        />
      </div>
    </el-main>
  </div>
</template>

<script>
import Card from '../Card/Card';
import { mapGetters } from 'vuex';
export default {
  name: 'SelectedContent',
  components: { Card },
  computed: mapGetters(['getCardsToShow']),
};
</script>

<style scoped>
.el-main {
  background-color: #ffffff;
  color: #333;
  text-align: center;
  line-height: 1.2;
  min-height: 100%;
  padding: 0;
  margin-top: 70px;
}

.el-main-flexContainer {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
}

.text {
  font-size: 14px;
}

.item {
  padding: 18px 0;
}

.box-card {
  width: 400px;
  margin-bottom: 10px;
}

.box-card span {
  display: inline-block;
  width: 100%;
}
</style>
