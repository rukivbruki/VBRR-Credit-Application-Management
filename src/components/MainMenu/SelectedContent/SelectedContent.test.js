import { mount } from '@vue/test-utils';
import SelectedContent from './SelectedСontent';

describe('SelectedСontent.vue render', () => {
  test('renders correctly', () => {
    expect(mount(SelectedContent).element).toMatchSnapshot();
  });
});
