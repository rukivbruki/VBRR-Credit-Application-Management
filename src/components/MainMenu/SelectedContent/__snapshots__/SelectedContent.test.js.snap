// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedСontent.vue render renders correctly 1`] = `
<el-main>
  
  <card-stub
    links="Текущие задачи,Смена пароля"
    name="Действия"
  />
  <card-stub
    links="Автоматизированные рабочие места,Филиалы/Точки обслуживания,Параметры точек обслуживания"
    name="Пользователи"
  />
  <card-stub
    links="Дизайн бизнес процессов,Справочник массовой обработки,Дизайнер вызова по расписанию"
    name="Процессы"
  />
  
</el-main>
`;
